app/src/util/tools.ts

```diff
@@ -472,3 +472,16 @@ export const onChangeMarketCurrency = (
     }
   }
 }
+
+/**
+ *  Gets initial display collateral
+ * If collateral is cToken type then display is the base collateral
+ * Else display is the collateral
+ */
+export const getInitialCollateral = (networkId: number, collateral: Token): Token => {
+  if (collateral.address === getWrapToken(networkId).address) {
+    return getNativeAsset(networkId)
+  } else {
+    return collateral
+  }
+}
```

Comment:
We should have a test for this in `tools.spec.ts`, same for all other tools functions